src/service/chat.js

```diff
@@ -271,6 +271,55 @@ const initChatFromPostAnon = async ({source, id}) => {
   }
 };
 
+const addMemberGroup = async ({channelId, memberIds}) => {
+  try {
+    const response = await api.post('/chat/group/add-members', {
+      channel_id: channelId,
+      members: memberIds
+    });
+    console.warn('response', JSON.stringify(response));
+    if (response.status === 200) {
+      return Promise.resolve(response.data);
+    }
+    return Promise.reject(response.data);
+  } catch (e) {
+    if (e?.response?.data?.message) return Promise.reject(e?.response?.data?.message);
+    return Promise.reject(e);
+  }
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `addMemberGroup` function seems to be well implemented. However, the use of `console.warn` on line 280 for logging response data is not recommended in production code as it can potentially expose sensitive information. Consider removing this or replacing it with a more secure logging solution.
```diff
-    console.warn('response', JSON.stringify(response));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->